src/components/Events/EventCard.js

```diff
@@ -0,0 +1,41 @@
+import { React } from 'react';
+
+import EventItem from './EventItem'
+import './EventCard.css'
+
+function EventCard (props) {
+
+    const data = props.eventData
+    console.log('props data:', data)
+
+    // const currentDate = new Date()
+    // console.log('currentDate:', currentDate)
+
+    // const isPast = (date) => {
+    //    if (date < currentDate) {
+    //     //    return true
+    //     console.log(date < currentDate)
+    //    }
+    // };
+
+    const isResultNull = (result) => {
+        if (result === null) {
+            return true
+        }
```

Comment:
it's always a good idea to return its opposite value (`false` here) on other cases. So we can ensure that this function will return either `true` or `false`. Otherwise, we need to consider `undefined` case too.